src/main/java/bio/terra/testrunner/common/utils/AuthenticationUtils.java

```diff
@@ -75,15 +61,9 @@ public static GoogleCredentials getDelegatedUserCredential(
    */
   public static GoogleCredentials getServiceAccountCredential(
       ServiceAccountSpecification serviceAccount, List<String> scopes) throws IOException {
-    if (serviceAccountCredential != null) {
-      return serviceAccountCredential;
-    }
-
-    synchronized (lockServiceAccountCredential) {
-      File jsonKey = serviceAccount.jsonKeyFile;
-      serviceAccountCredential =
-          ServiceAccountCredentials.fromStream(new FileInputStream(jsonKey)).createScoped(scopes);
-    }
+    File jsonKey = serviceAccount.jsonKeyFile;
+    GoogleCredentials serviceAccountCredential =
```

Comment:
nit: you can return this directly, it doesn't need to be a local var

Reply:
Removed the unnecessary local variables.